src/test/resources/com.scottlogic.features/Application.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Application Exists
```

Comment:
again, should be part of Geidre's PR , and can be an actual test of the User endpoint